README.md

```diff
@@ -17,15 +17,16 @@ python3 setup.py install --user
 pip3 install standup-notes
 ```
 ### Bash Setup
-For a 1 time execution, use
+
+To have bash completion, find where the ```standup-notes.bash``` file is in your system by running the following command
 ```
-source /path/to/standup-notes.bash
+pip3 show standup-notes
 ```
-To permanently have bash completion, copy the standup-notes.bash into the bash-completion.d folder in your system
+Under the ```Location: ``` tab, this is where the resoruces folder is located which contains ```standup-notes.bash```.
+Run the following command to copy the bash completion script to the correct directory
 ```
-cp /path/to/standup-notes.bash /path/to/bash_completion.d
+cp /path/to/resources/standup-notes.bash /path/to/bash_completion.d/
```

Comment:
Could you add instructions on where to find the bash_completion.d folder?

Reply:
Just to confirm, just add this?
```
pip3 show standup-notes
```

Reply:
No, that would show where to find the standup-notes.bash script. I mean the folder where the bash script should be moved to, (/path/to/bash_completion.d/). I'm not sure what that is or how it's used.

Reply:
Ah, folder that it should be moved to is /etc/bash_completion.d/ which is now in readme.md